src/components/result.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly remove the `eslint-disable` statement from this file. `Eslint` helps us to write clean, readable and industry-standard codes. Using `eslint-disable` statements defeats this purpose. Kindly remove it. 👍 💻